### DISCUSSION TITLE
카테고리 별로 '공개설정', '목록스타일', '대표이미지', '카테고리소개' 설정 반영.

### DISCUSSION THREAD

**Discussion by PulpExposed** [pronist/hELLO#245] [Ideas]
![스크린샷 2024-07-22 오후 10 22 49](https://github.com/user-attachments/assets/62ba4f6b-60ad-45a6-902c-8d147d8ac2b0)

위와 같은 티스토리의 기본 카테고리 관리 내역들이 스킨에 반영되면 좋겠습니다.

**Comment by pronist** [comment]
@PulpExposed 안녕하세요.

**카테고리 공개/비공개**, **목록 스타일** 은 지금도 적용됩니다. 카테고리 공개 여부의 경우 블로거 본인이 아닌 **타인** 을 대상으로 하므로 로그아웃을 한 이후 테스트해보셔야 한다는 점 잊지마시기 바랍니다. 카테고리 이미지와 설명의 경우 지금과 같이 추가 요청과 과거 삭제 요청이 동시에 발생하고 있어 [4.7.0](https://github.com/pronist/hello/releases/tag/4.7.0a) 버전에 삭제를 한 바 있습니다. 이를 같은 방식으로 되돌릴 예정은 없습니다. 단, 삭제 요청과 추가 요청 두 요구를 모두 충족시킬 수 있는 더 나은 방법이 제시되거나 떠오르면 그때 개선할 예정입니다.
